nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -92,8 +93,8 @@ contract StakingEscrow is Issuer {
         uint16 lastActivePeriod;
         bool measureWork;
         uint256 completedWork;
+        bool windDown;
 
-        uint256 reservedSlot1;
```

Comment:
awesome to see this in use :)
BTW, does it make sense to keep an indication that the reservedSlot was of size 32B? I mention this in case a new `bool` field is added and that perhaps can still fit into the previous slot.

Reply:
Now that I think of, that depends on the size of all the previous slots, and it can become difficult to track. Nevermind then.

Reply:
I could add comment that last slot with bool has some empty space (previous uint256 and next slot also uint256, so we have room for new value)

Reply:
Great, maybe it can be useful in the future :)

Reply:
✔️